src/main/flight/pid.c

```diff
@@ -278,19 +278,22 @@ void pidResetIterm(void)
 
 void pidUpdateTpaFactor(float throttle)
 {
-    static bool isTpaLowFaded = false;
     // don't permit throttle > 1 & throttle < 0 ? is this needed ? can throttle be > 1 or < 0 at this point
     throttle = constrainf(throttle, 0.0f, 1.0f);
-    bool isThrottlePastTpaLowBreakpoint = (throttle < pidRuntime.tpaLowBreakpoint && pidRuntime.tpaLowBreakpoint > 0.01f) ? false : true;
+
+    const bool tpaLowInactive = (pidRuntime.tpaLowDisabled || (throttle >= pidRuntime.tpaLowBreakpoint));
     float tpaRate = 0.0f;
-    if (isThrottlePastTpaLowBreakpoint || isTpaLowFaded) {
+    if (tpaLowInactive) {
+        // to get here, throttle exceeded the tpaLow threshold
```

Comment:
(not exactly - it may be disabled because of pidProfile->tpa_low_breakpoint < 1010)

Reply:
Indeed, will update the comment